contrib/vagrant/provision-util.sh

```diff
@@ -128,6 +128,9 @@ os::provision::set-os-env() {
   fi
   os::provision::set-bash-env "${origin_root}" "${config_root}" \
 "/root/${file_target}"
+
+  # Make symlinks to the bash completions for the openshift commands
+  ln -s ${origin_root}/contrib/completions/bash/* /etc/bash_completion.d/
```

Comment:
Ignore this....
On Fedora 21, re-installing bash-completion pkg and then logout and login resolved the issue.
Looks like all files under /etc/bash_completion.d is sourced by default when the user logs in.